src/components/DataTable/DataTable.tsx

```diff
@@ -93,14 +97,50 @@ const DataTablePresentationComponent = memo(
 
         const rowCount = rowCountProp || records.length;
 
+        const getVisibleColumnIndices = useCallback(
+            (x: number = 0) => {
+                const columns = store.current.columns || [];
+                const visibleColumns: number[] = [];
+                const _containerWidth = store.current.containerWidth || 0;
```

Comment:
need to check columns.

If columns are hidden, then we'll have incorrect indices and possibly blanks

Reply:
Same with column widths